sites/all/themes/orwell/sass/component/_search.scss

```diff
@@ -155,6 +154,20 @@ $icon-color: #9c9c9c;
     height: $item-height;
     align-items: center;
 
+    fieldset {
+      background-color: transparent;
+      border: 0;
+      padding: 0;
+      margin: 0;
+      min-width: 0;
+
+      legend {
+        position: absolute !important;
+        top: -9999px !important;
+        left: -9999px !important;
+      }
+    }
+
```

Comment:
Is there an existing class like bootstraps `sr-only`? Otherwise maybe we should consider creating one?